synergy-design-system/synergy-design-system
# docs: 📚 Document used typographic styles and fonts

# Pull Request

## 📖 Description

This PR adds `Open Sans` as a dependency to our demo (via external npm package) and makes sure the font is also used in Storybook (using the officially supported archive from SDS).

It also adds documentation as described in #615.

### 🎫 Issues

Closes #614
Closes #615 

## 👩‍💻 Reviewer Notes

Just play around with Storybook and the framework demos (e.g. inspect loading via Network-Panel, check if correct font is used, check dist versions).

## 📑 Test Plan

- Check out this branch locally (mandatory!)
- Run `pnpm i -r` (or `pnpm clean` in the repository root)
- Start Storybook. Notice the new font in contrast to the current one.
- Check that all components use the correct font in Storybook
- Move to `packages/_private` and inspect the same for the framework demos
- Let the tests start via `cd packages/_private/e2e-demo-test && pnpm start`. There should not be any errors.

## ✅ DoD
